content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md

```diff
@@ -24,7 +24,7 @@ For development purposes Ripple recommends running `rippled` as your own user, n
 
         $ brew install git cmake pkg-config protobuf openssl ninja
 
-0. Install Boost 1.67.0. `rippled` 1.2.0 is compatible with Boost 1.67. <!--#{ no boost@1.67 formula, so must manually compile and install. will eventually upgrade to boost 1.68, but needs testing first }# -->
+0. Install Boost 1.67.0 or higher. `rippled` 1.3.0 is compatible with Boost 1.67.0 and higher. The latest version of Boost available in the Homebrew repositories isn't new enough, so you must install Boost manually:
```

Comment:
Might want to take the word "latest" out of this sentence:

The latest version of Boost available in the Homebrew repositories isn't new enough, so you must install Boost manually:

Reply:
I see where you're coming from, but Homebrew actually contains several versions of Boost, none of which are new enough.